docs/pages/guides/CssThemeSpacing.example.jsx

```diff
@@ -1,19 +1,18 @@
 import React, { useState } from 'react';
-import { createMuiTheme } from '@material-ui/core';
-import { ThemeProvider } from '@material-ui/styles';
 import { DateTimePicker } from '@material-ui/pickers';
+import { createMuiTheme, MuiThemeProvider } from '@material-ui/core';
 
-const defaultMaterialTheme = createMuiTheme({
+const muiTheme = createMuiTheme({
   spacing: 2,
```

Comment:
Should this be a visual regression test rather than a demo? But I'm not sure a visual test is needed. On the main repository, we had very few bug report / concern on this end. In https://material-ui.com/customization/density/#explore-theme-density, people can tweak the density.

Reply:
Actually there were some issues with it in pickers #1128. So this is more like a proof that we are not completely broken with custom spacing. But I agree that it needs to be a demo